Api/csharp/ArmoniK.Api.Client/Submitter/SubmitterClientExt.cs

```diff
@@ -225,17 +180,52 @@ await stream.RequestStream.CompleteAsync()
                                      DataComplete = true,
                                    },
                    };
+    }
 
-      if (isLast)
+    /// <summary>
+    ///   Get result without streaming
+    /// </summary>
+    /// <param name="client">gRPC client to the Submitter</param>
+    /// <param name="resultRequest">Request for result</param>
+    /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
+    /// <returns>
+    ///   A stream in which the data will be written
+    /// </returns>
+    /// <exception cref="Exception">a result reply chunk is not data, rending it impossible to reconstitute the data</exception>
+    /// <exception cref="ArgumentOutOfRangeException">result reply type is unknown</exception>
+    [PublicAPI]
+    public static async Task<MemoryStream> GetResultAsStreamAsync(this gRPC.V1.Submitter.Submitter.SubmitterClient client,
```

Comment:
I dont think it works as expected. From what I understand, you create and populate the stream entirely before returning it.